Add Frontier (FRONT) Space

philiparthurmoore opened the pull request:
**Add space for Frontier**  
  
Etherscan: https://etherscan.io/token/0xf8C3527CC04340b208C854E985240c02F7B7793f 
CoinGecko: https://www.coingecko.com/en/coins/frontier
CMC: https://coinmarketcap.com/currencies/frontier/
Website: https://frontier.xyz/ 
Twitter: https://twitter.com/FrontierDotXYZ
Telegram: https://t.me/FrontierDotXYZ

gokaiorg commented on skins/frontier.scss:
```diff
@@ -0,0 +1,19 @@
+  --header-bg: #f8eee4;
+  --block-bg: transparent;
+}
+#app.frontier .state,
+#app.frontier .State {
+  color: #2a1f21;
+  background-color: #e9e8e8;
+}
+#app.frontier .d-block .bg-purple {
+  color: #fff!important;
+  background-color: #ad6037!important;
```
Please use only theme variables. No extra custom styles are allowed.

philiparthurmoore replied:
This has been addressed.

gokaiorg commented on spaces/frontier/index.json:
```diff
@@ -0,0 +1,51 @@
+  "symbol": "FRONT",
+  "skin": "frontier",
+  "domain": "vote.frontier.xyz",
+  "strategies": [
```
There is a problem with your strategies, your space doesn't load properly. You could use only erc20-balance-of for now or ask @bonustrack for help.

philiparthurmoore replied:
I will take out the Balancer and Uniswap strategies however is there a way to test this locally to see what the issue may be? I ran linting on everything and nothing seems strange however if for now only the balance strategy will work then I will use that one until I figure out the LP issue.

philiparthurmoore replied:
I have removed the Balancer strategy and kept the Uniswap strategy. Can you please tell me if it's okay now?

philiparthurmoore replied:
I believe the reason this wasn't working was the symbol naming. I have named all strategies `FRONT` now, which is in line with the token symbol name.

philiparthurmoore replied:
Sorry to spam. I believe this is a syntax issue that I am now fixing. I looked at https://raw.githubusercontent.com/snapshot-labs/snapshot-spaces/f283034e7e5365652c0f5a28fbff6ee83509416d/spaces/aragon/index.json for reference.

philiparthurmoore replied:
All syntax issues committed and addressed here: https://github.com/snapshot-labs/snapshot-spaces/pull/498#issuecomment-728643848

gokaiorg replied:
Ok everything load well for me, i let @bonustrack confirm we are all good with this strategies.

philiparthurmoore commented:
I have fixed the strategy issues. I believe it was due to syntax errors which I have fixed. 

```json
{
  "name": "Frontier",
  "network": "1",
  "symbol": "FRONT",
  "skin": "frontier",
  "domain": "vote.frontier.xyz",
  "strategies": [
      {
        "name": "erc20-balance-of",
        "params": {
          "address": "0xf8C3527CC04340b208C854E985240c02F7B7793f",
          "symbol": "FRONT",
          "decimals": 18
        }
      },
      {
        "name": "balancer",
        "params": {
          "address": "0xf8C3527CC04340b208C854E985240c02F7B7793f",
          "symbol": "FRONT BPT"
        }
      },
      {
        "name": "uniswap",
        "params": {
          "address": "0xf8C3527CC04340b208C854E985240c02F7B7793f",
          "symbol": "FRONT LPT"
        }
      }
    ],
  "members": [
    "0xfC43f5F9dd45258b3AFf31Bdbe6561D97e8B71de",
    "0xDC36158CDb5A9bb63dbe07F10E7965dE9E802691",
    "0xeE32B4336183a1299C13d738b93c64c2FcdAa3C8",
    "0x756B70E09eE63B05D9b697eCDB4c5aE68F8f8f34",
    "0x347A40aA8E9D4c4f9E676C2A2FF846565892e3a3",
    "0xF0C4E92Fbb7859D509Aa624518EaC11FAf00136d",
    "0xb4A231260D01731fd077f234d8f7A2A1455F36a2",
    "0x8c7C82f45A1910280fDe0a20268e36E67f0cccD0",
    "0x7773e5b161aabe917a574Bc65529Ee46CF9897C8",
    "0x5F422FDdAD5e72E28a18dC147A44CaB279d0D368"
  ],
  "filters": {
    "defaultTab": "all",
    "minScore": 1000,
    "onlyMembers": false,
    "invalids": []
  }
}
```
